panel/package.json

```diff
@@ -5,7 +5,9 @@
   "license": "BSD-3-Clause",
   "repository": {},
   "dependencies": {
-    "bokehjs": "^1.3.4"
+    "@types/gl-matrix": "^2.4.5",
+    "bokehjs": "^1.3.4",
+    "gl-matrix": "^3.1.0"
```

Comment:
I wasn't aware that you could actually load dependencies like this yet. What ends up loading the dependency?

Reply:
I suppose node and the typescript compiler to create the .js